docs/user_interface/how_to_use_ui.rst

```diff
@@ -0,0 +1,154 @@
+=========================
+How To Use User Interface
+=========================
+
+.. _how-to-use-ui-page:
+
+Step-by-step guide
+------------------
+
+.. container::
+
+   .. container::
+
+      .. container::
+
+         **Step 1.** Download the latest version of PARETO for your
+         operating system at the `Project PARETO Website <https://www.project-pareto.org/software>`_.
+
+         .. container:: screenshot-div
+
+            |software-website|
+
+         **Step 2.** Follow the installation steps to install the
+         application. For Windows, see below:
+
+         .. container:: screenshot-div
+
+            |windows-installer|
+
+         **Step 3.** Run the application. You should see a splash page
+         before being redirected to the scenarios page:
+
+         .. container:: screenshot-div
+            
+            |scenarios-page|
+
+         **Step 4.** Click on "Create new scenario". Download from one
+         of the available sample inputs, or use your own data, then name the scenario and click
+         "Create scenario":
+
+         .. container:: screenshot-div
+
+            |create-scenario|
+
+         **Step 5.** Browse the input plots and tables, with the option
+         to edit any of the inputs:
+
+         .. container:: screenshot-div
+
+            |input-page|
+
+         **Step 6.** Click "CONTINUE TO OPTIMIZATION" button, and choose
+         from the optimization settings:
+
+         .. container:: screenshot-div
+
+            |model-settings|
+
+         **Step 7.** Click "OPTIMIZE" to run the PARETO strategic model:
+
+         .. container:: screenshot-div
+
+            |running-model|
+
+         **Step 8.** When the model is finished running, the KPI
+         dashboard will be visible from the model results section:
+
+         .. container:: screenshot-div
+
+            |kpi-dashboard.|
+
+         **Step 9.** Analyze the output by looking through tables and
+         graphs, including the Sankey diagram that interactively
+         displays the transport of produced water (via piping or trucking):
+
+         .. container:: screenshot-div
+
+            |sankey-diagram|
+
+         **Step 10.** If you are unhappy with any of PARETO's
+         infrastructure decisions, feel free to override with your own
+         decisions in the "infrastructure buildout" tab:
+
+         .. container:: screenshot-div
+
+            |infrastructure-override|
+
+         **Step 11.** Compare different scenarios by clicking on any of
+         the compare icons from the scenario page:
+
+         .. container:: screenshot-div
+
+            |scenario-comparison|
+
+
+.. _map_example:
+
+Map Feature (Beta)
+------------------
+
+.. container::
+
+   The UI team has implemented a method for users to interact with the UI by uploading a map file representing their network. 
```

Comment:
Instead of "The UI team has implemented for users to .." can say "Users can ..." eg. "Users can upload a map file to the UI that represents their network"